data.sql

```diff
@@ -6,4 +6,12 @@ VALUES
   (2, 'Gabumon', '2018-11-15', 2, TRUE, 8),
   (3, 'Pikachu', '2021-01-07', 1, FALSE, 15.04),
   (4, 'Devimon', '2017-05-12', 5, TRUE, 11);
-
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg, species) 
+VALUES 
+(5,'Charmander', '2020-02-08', 0, false, -11,'unspecified'),
+(6,'Plantmon', '2021-11-15', 2, true, -5.7,'unspecified'),
+(7,'Squirtle', '1993-04-02', 3, false, -12.13,'unspecified'),
+(8,'Angemon', '2005-06-12', 1, true, -45,'unspecified'),
+(9,'Boarmon', '2005-06-07', 7, true, 20.4,'unspecified'),
+(10,'Blossom', '1998-10-13', 3, true, 17,'unspecified'),
+(11,'Ditto', '2022-05-14', 4, true, 22,'unspecified');
```

Comment:
- [ ] From [the project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), the data that suppose to be inserted into the `animals` table has no values for the species column which means the species columns suppose to be null in the `animals` table :blush: but you set the value to `unspecified` :relaxed: